### DISCUSSION TITLE
N64 internal mod

### DISCUSSION THREAD

**Discussion by miguel92** [darthcloud/BlueRetro#239] [Q&A] [answered]
Hello everyone, I want to do and internal installation of the ESP32 on my N64 and I have a question:
- Can you still use wired controllers on the N64 ports while the ESP32 is powered on? I don't know if it possible or if it is possible could this be dangerous for the system?

**Comment by pkerga** [comment]
I have an internal installation 

https://twitter.com/pkerga22/status/1496542625654095877?t=cu151OdhewUnjKScyjgCqQ&s=19

https://twitter.com/pkerga22/status/1500483174790569989?t=CVQovzlRA97EXC-mDdpnLQ&s=19

and wired controllers still work.

**Reply by miguel92** [reply]
Yeah thank you so much for your help :)

**Comment by pkerga** [comment]
Another vidéo
https://youtu.be/0vbnkY_3Q6o

**Comment by darthcloud** [comment, marked as answer]
BTW this pure luck if it work. Results may vary base on various factors.

My official stand is that you can't use BlueRetro with original controller simultaneously. You either need switch on ESP32 power or on individual data lines.

**Reply by miguel92** [reply]
Mmm that is what I was thinking. I saw that video too but I not sure if it can damage the system or something like that. So you recommend to install a switch to turn on/off the ESP32 if you want to use an original controller.

**Reply by darthcloud** [reply]
Don't worry it wont damage anything, I'm just saying it might not work gameplay wise.

**Reply by miguel92** [reply]
Perfect then, but I will install a switch just in case. Thank you so much, BlueRetro is awesome!

**Reply by persechini** [reply]
I'd like to ad to this one. I had one wired controller that wasn't working when connecting to port 1, but would work in other ports, but my other 2 controllers would work no problem in port one, as long as I didn't have any wireless controller connected to bluerretro

The one that didn't was a black controller bought standalone and the PCB had a sticker for the brazilian distributor. The other 2 that worked on were the one bundled with my console and one bundled with a friend's console that I bought from him at one point in time

At a glance I didn't see any difference between the PCBs, so I don't know what would be the reason they behave differently
